vllm/sampling_params.py

```diff
@@ -184,6 +184,9 @@ def __init__(
 
         self._verify_args()
         if self.use_beam_search:
+            # Lazy import to avoid circular imports.
+            from vllm.usage.usage_lib import set_runtime_usage_data
+            set_runtime_usage_data("use_beam_search", True)
```

Comment:
IIUC, we don't track the number of beam search requests or its ratio, but track the cases that the server receives at least one beam search request, right?

Reply:
Correct. I think in the end I want to get to some understanding of "% of vLLM deployments using beam search"